smac/utils/io/traj_logging.py

```diff
@@ -70,6 +69,7 @@ def __init__(self, output_dir, stats):
                         '"Configuration..."\n')
 
             self.aclib_traj_fn = os.path.join(output_dir, "traj_aclib2.json")
+            self.alljson_traj_fn = os.path.join(output_dir, "traj_alljson.json")
```

Comment:
json.json seems to be a bit redundant

Reply:
But isn't "traj_all.json" a bit ambigous? Do you have any proposals for a title of the format?

Reply:
I think `traj.json` would suffice and is not ambigous at all.